### ISSUE TITLE
[Feature Request] Tabs

### ISSUE THREAD

**Issue by ralcaidev** [wavebox/waveboxapp#885]
* Wavebox Version: 4.5.5
* Operating System & Version: Any
* Account type (if applicable): Any

I think it would be a great idea to add tabs, as seen in Station, for each service.
If you right click a URL -> Open Link with Wavebox, you have three options: 

1. New Window.
2. New Service.
3. Here (which is the worst since there's no "back" button like web browsers or mouse listener to back button).

For me, the best solution was:
1. Let the user choose the default behaviour when you single-click on a URL.
2. Create "tabs" like Station: When you select a service, there are "recent, pinned and tasks", place it there by adding "tabs".

What do you guys think?

**Comment by Thomas101** [comment]
Yeah definitely agree! The Recent, Pinned and Tasks are just the first step in a bunch of new tools we're adding.

The New Window that Wavebox has at the moment, works for some use-cases (e.g. editing one or two Google Docs), but not when you have lots of open windows. It just gets messy.

Stay tuned :)

**Comment by apidevlab** [comment]
Another plus 1 👍

**Comment by markrakozy** [comment]
Yet another vote. Using Google Drive. Want to open GooglsDocs documents from there in a tab. "Within" the Google Docs service.

**Comment by benvblanco** [comment]
> Yet another vote. Using Google Drive. Want to open GooglsDocs documents from there in a tab. "Within" the Google Drive service.

Another plus 1 👍 . Ran into this scenario today.

**Comment by cantchooseaname8** [comment]
Count me in for this too!
